tests/operator_tests.py

```diff
@@ -453,6 +453,33 @@ def equality_helper(a, n):
             equality_helper(ak_edge_cases // a_vect, np_edge_cases // n_vect)
             equality_helper(a_vect // ak_edge_cases, n_vect // np_edge_cases)
 
+    def test_pda_sqrt(self):
+        n = np.array([4, 16, -1, 0])
+        a = ak.array(n)
+
+        s = ak.sqrt(a)
+        ex = np.sqrt(n)
+
+        # id nans
+        nan = ak.isnan(s)
+        npnan = np.isnan(ex)
+        self.assertListEqual(nan.to_list(), npnan.tolist())
+
+        # check results are equal remove nans
+        self.assertListEqual(s[~nan].to_list(), ex[~npnan].tolist())
+
+    def test_pda_power(self):
+        n = np.array([10, 5, 2])
```

Comment:
yeah i think testing some of the weirder edge cases would be good, maybe `inf`, `-inf`, `0.0`, and `-0.0`

Reply:
Added